lib/spack/spack/ci.py

```diff
@@ -547,11 +614,13 @@ def release_jobs(parser, args):
 
                 job_dependencies = []
                 if spec_label in dependencies:
-                    job_dependencies = (
-                        [get_job_name(phase_name, strip_compilers,
-                                      spec_labels[dep_label]['spec'],
-                                      osname, build_group)
-                            for dep_label in dependencies[spec_label]])
+                    for dep_label in dependencies[spec_label]:
```

Comment:
can this be simplified with:

```python
for dep_label, something in dependencies['spec_label'].items():
```

?

Reply:
I'm having a hard time coming up with the `something` that makes this loop simpler.  Hopefully it's ok the way it is.